homa.h

```diff
@@ -83,25 +83,28 @@ struct homa_send_args {
 	/** @dest_addr: Address of server to which request will be sent. */
 	sockaddr_in_union dest_addr;
 
-	uint32_t _pad1;
+	uint32_t flags;
```

Comment:
We can talk about this if you like, but I'm still not seeing a compelling reason to define fields for which there is no existing use. A pad field seems like the right thing here.